src/js/components/CharityPage.jsx

```diff
@@ -44,6 +45,18 @@ const CharityPage = () => {
 	if ( ! charity) {
 		return <Misc.Loading pv={pvCharity} />;
 	}
+	let impact;
+	let project = NGO.getProject(charity);
+	if (project) {
+		let targetCount = DataStore.getValue([
```

Comment:
if in the course of implementing this you now understand what 'targetCount' means, please add a comment! It's non-obvious to me.

Reply:
Half done - comment added to code. I can tell you what it is -- a setting for "how many units should we give a costing for?" -- But I can't remember why we wanted that flexibility.